lib/components/Utilities/Icon/index.tsx

```diff
@@ -15,24 +15,31 @@ const iconVariants = cva("inline-block fill-current", {
       sm: "h-4 w-4",
       xs: "h-3 w-3",
     },
+    color: {
+      primary: "text-foreground",
+      secondary: "text-secondary-foreground",
+      tertiary: "text-primary-foreground",
+    },
```

Comment:
Do we need these properties? I mean they don't hurt, but we default to using whatever color the container has. I guess if you don't set the property, it will just not add that class and still have this behavior?

Reply:
I can get rid of these, but I have 2 icons in the same container with different colors, so that forces me to wrap them and get it a bit dirtier, I'll push it so you can see what I mean.